Bottom: cc81f8c284aa6a261a5e6761f6b03d1166400ef3
Top:    1eb9153d8c6542bcaadb5b40c8663f0e285967b6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-05-17 14:58:11 +0100

Refresh of fix-build-warnings-in-gtkhid

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 4ad2fb2..dfbab22 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1738,7 +1738,7 @@ GhidDrawMask (int side, BoxType * screen)
 
   gui->graphics->use_mask (HID_MASK_OFF);
 
-  first_run = false;
+//  first_run = false;
 }
 
 static void
